pkg/koordlet/statesinformer/states_device_linux.go

```diff
@@ -119,6 +131,17 @@ func (s *statesInformer) updateDevice(name string, gpuDevices []schedulingv1alph
 		device.Spec = schedulingv1alpha1.DeviceSpec{
 			Devices: gpuDevices,
 		}
+
+		if device.Labels == nil {
```

Comment:
It is necessary to pay attention to the existing scenes. If device.Spec.Devices does not change, the following label will not be updated.

Reply:
+1，the DeepEqual on L126

Reply:
fixed